### PR TITLE
Prevent Vue from Loading Variables Before Evaluation

### PR DESCRIPTION
**Pull request by montana-mil**
## Description
When loading tables, charts, or longer lists, some Vue markup was flashing on screen. Using `v-cloak` on these Vue components prevents that behavior so that nothing loads until all the variables have been evaluated. 

## Pivotal Tracker
https://www.pivotaltracker.com/story/show/162622309
https://www.pivotaltracker.com/story/show/161959592

### CODE REVIEW

File: templates/audit_log/audit_log.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1 class="block-list__title">Activity History</h1>
       {% endfor %}
     </ul>
 
-  </section>
+    {{ Pagination(audit_events, 'atst.activity_history') }}
 
-  {{ Pagination(audit_events, 'atst.activity_history') }}
+  </section>
```
Comment (dandds) [inline templates/audit_log/audit_log.html]: It looks like moving this changes the spacing above it. Does it render badly before the Vue component is ready if it's outside the `section`?

Comment (montana-mil) [reply templates/audit_log/audit_log.html]: it just shows the pagination before anything else, so it looks funny. do you think theres another way?

Comment (dandds) [reply templates/audit_log/audit_log.html]: You could wrap the pagination in a blank div where it is and put `v-cloak` on the div, maybe?

Comment (montana-mil) [reply templates/audit_log/audit_log.html]: ok, can do!

Comment (leigh-mil) [reply templates/audit_log/audit_log.html]: Caaaaan doooooo!

**Pull request merged**